CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+
+If you find a bug in the source code, an issue with an implementation, or a mistake in the documentation, you can help by submitting an issue in [GitHub Issues][githubissues] for this [repository](https://github.com/Semantic-Org/Semantic-UI-Angular/issues).
+
+### <a name="submitting"></a> Submitting an Issue
+
+Please search through [existing issues][githubissues] and the archive first to see if your question was already answered or if the issue you are experiencing has already been reported. We need your help to not duplicate issues so we can focus on fixing issues and working on new features.
+
+Once you have determined that you have an unreported problem, please use the following guidelines when opening an issue.
+
+- Overview of the Issue - Include any non-minified stack trace and other runtime errors that are available and relevant to the issue.
+- Use Case - explain why this is a bug for you
+- Angular, Semantic UI, and Semantic UI Angular Version(s) - Is this an issue with Angular, the Semantic UI CSS framework, or a regression issue with this module?
+- Browsers and Operating System - Is this a problem with all browsers, or specific browsers and/or specific OS? Also, be aware that browser requirements for Angular and Semantic UI apply and are the baseline requirements for this module.
+- Reproduce the Error - Please, please, please provide a live example (using Plunker or JSFiddle) or an unambiguous set of steps! Doing the footwork up front will allow us to replicate and fix the problem faster.
+- Related Issues - Please note any related issues, if any.
+- Suggest a Fix - If you have an idea of what's causing the problem (line of code or a commit), please point it out.
+
+## <a name="newfeatures"></a> Requesting Features
+
+You can submit an issue in [GitHub Issues][githubissues]. If you would like to implement it yourself, then consider the scope of the change:
+
+- **Major Change**: If you are interested in a major change, please [get in touch](#getting-help) with the team so we can coordinate our efforts, avoid duplicate work, and ensure your contributions are successfully accepted into the project.
+- **Minor Change**: If the change is small, you can fork the [repository][github] and create a Pull Request with your changes. See [Submitting a Pull Request](#pullrequest) for more information.
+
+## <a name="commiting"></a> Git Commit Guidelines
+
+We follow the `conventional-changelog` format used by the AngularJs team. You can follow the format manually, but we highly recommend you use a CLI wizard such as [Commitizen][commitizen].
+
+In fact, we have configured this repository to work with Commitizen. First, ensure Commitizen CLI is installed via `npm install -g commitizen`. Next, make sure the clone of this repository is installed via `npm install`. You're all set! Now, whenever you want to commit, just stage your changes e.g. `git add .` and commit using `git cz`. Follow the prompts and you're done.
+
```

Comment:
Add a line about unit tests, for users before committing should run unit tests and fix linting warnings if there are any.